### PR TITLE
feat: implement stack

### PR DESCRIPTION
**Pull request by ArielRojas**
image

### CODE REVIEW

File: DataStruturesPartI/MyStack.cs [hunk 1]
```diff
@@ -0,0 +1,46 @@
...
+
+        T value = _top.Data;
+        _top = _top.Next;
+
+        return value;
+    }
+
+    public bool IsEmpty()
+    {
+
```
Comment (bernarth) [inline DataStruturesPartI/MyStack.cs]: Here you have unnecessary blank line

**Review by bernarth — COMMENTED** [review]
Good job fix the comments

File: DataStruturesPartI/MyStack.cs (hunk 1 above)
Comment (ArielRojas) [reply DataStruturesPartI/MyStack.cs]: Ok, I removed it.

**Comment by bernarth** [comment]
You did an excellent work. Angel won resolved his comments first. Great job anyway keep on doing you have 10/10

**Pull request closed without merging**